src/app/main/homepage/homepage.component.ts

```diff
@@ -13,39 +16,51 @@ import { PublicService } from '../../core/api/services/public.service';
 })
 export class HomepageComponent implements OnInit {
   public totalSite: number = 0;
-  public lastSiteRegister!: SiteDto;
+  public lastFiveSite: SiteListDto[] = [];
   public totalRoute: number = 0;
-  public lastRouteRegister!: RouteDto;
+  public lastFiveRoute: RouteListDto[] = [];
   public totalUsers: number = 0;
   public loaded: boolean = false;
+  public lastFiveCheckedRoute: RouteListDto[] = [];
+  public sitesOptions: TableSiteOptions;
   constructor(
     private readonly publicService: PublicService,
-    private readonly messageService: MessageService
-  ) {}
+    private readonly messageService: MessageService,
+    private readonly languageService: LanguageService
+  ) {
+    this.sitesOptions = {
+      loading: true,
+      forAdmin: false,
+      fullView: false,
+    };
+  }
   ngOnInit(): void {
     this.loadData();
   }
 
-  private loadData() {
+  /**
+   * Charge les donnee publique de la homepage
+   * @private
+   */
+  private loadData(): void {
     this.publicService.publicControllerGetDataForHomePage().subscribe({
       next: data => {
-        console.log(data);
+        this.lastFiveCheckedRoute = data.lastFiveCheckedRoutes;
         this.totalSite = data.totalSites;
-        this.lastSiteRegister = data.lastSite;
+        this.lastFiveSite = data.lastFiveSite;
         this.totalRoute = data.totalRoutes;
-        this.lastRouteRegister = data.lastRoute;
+        this.lastFiveRoute = data.lastFiveRoute;
         this.totalUsers = data.totalUsers;
+        this.loaded = !this.loaded;
```

Comment:
?

Reply:
condition d'affichage de template html a la difference des this.loading la je veux savoir que toute les souscrption sont fini et que j'ai bien un status laoded a true